### PR TITLE
Glatos touch up

### PR DESCRIPTION
**Pull request by SoftwareMonk**
All the modifications needed to make this workshop ACT correctly :sunglasses:

### CODE REVIEW

File: _episodes/07-introduction-to-glatos.md [hunk 1]
```diff
@@ -23,41 +23,65 @@ setwd("./data")
...
+With our new file in hand, we'll want to use the read_otn_detections function
+to load our data into a dataframe. In this case, our data is formatted in the ACT
+style- if it were GLATOS formatted, we would want to use read_glatos_detections()
+instead.
+
 Remember: you can always check a function's documentation by typing a question
 mark, followed by the name of the function.
 ~~~
-## glatos help files are helpful!! ####
+## GLATOS help files are helpful!! ####
```
Comment (naomitress) [inline _episodes/07-introduction-to-glatos.md:82]: should be glatos

File: _episodes/08-more-features-of-glatos.md [hunk 2]
```diff
@@ -52,7 +107,10 @@ ATTdata$Station.Information
 ~~~
 {: .language-r}
 
-And then you can use your data with the VTrack package. You can call its abacusPlot function to generate an abacus plot:
+And then you can use your data with the VTrack package. You'll notice that not all the detections made it into the ATT object. That's because the conversion function only keeps detections that it has station metadata for. Tags with no detections will also be thrown out by the function. This is to prevent issues
+with VTrack.
+
+You can call its abacusPlot function to generate an abacus plot:
 ~~~
 # Now that we have an ATT dataframe, we can use it in VTrack functions:
```
Comment (naomitress) [inline _episodes/08-more-features-of-glatos.md:116]: is there something missing here?

File: _episodes/08-more-features-of-glatos.md [hunk 3]
```diff
@@ -31,19 +51,54 @@ Both of these methods are similar and will almost always give different results,
 explore them all to see what method works best for your data.
 
 
-glatos strives to be interoperable with other scientific R packages. Currently, we can
-crosswalk our data over to the package [VTrack](https://github.com/RossDwyer/VTrack). Here's an example:
+
+glatos strives to be interoperable with other scientific R packages. Currently, we can 
+crosswalk GLATOS data over to the package [VTrack](https://github.com/RossDwyer/VTrack).
```
Comment (naomitress) [inline _episodes/08-more-features-of-glatos.md:56]: GLATOS should be glatos

File: _episodes/08-more-features-of-glatos.md [hunk 4]
```diff
@@ -8,21 +8,41 @@ questions:
 
 glatos has some more advanced analytic tools beyond filtering and creating events.
 
+
```
Comment (naomitress) [inline _episodes/08-more-features-of-glatos.md:11]: title and questions need to be changed to glatos

File: _episodes/09-basic-visualizations.md [hunk 5]
```diff
@@ -18,13 +18,11 @@ against the appropriate stations.
 # ?glatos::abacus_plot
```
Comment (naomitress) [inline _episodes/09-basic-visualizations.md:18]: title and question should be changed to glatos

File: code/day2/GlatosCode.R [hunk 6]
```diff
@@ -7,15 +7,43 @@ library(glatos)
 library(tidyverse)
```
Comment (naomitress) [inline code/day2/GlatosCode.R:7]: title should be changed to glatos

File: code/day2/GlatosCode.R [hunk 7]
```diff
@@ -7,15 +7,43 @@ library(glatos)
...
+detections <- tibble()
+for (detfile in list.files('.', full.names = TRUE, pattern = "proj.*\\.zip")) {
+  print(detfile)
+  tmp_dets <- read_csv(detfile, col_types = format)
+  detections <- bind_rows(detections, tmp_dets)
+}
+write_csv(detections, 'all_dets.csv', append = FALSE)
+
 
 ## GLATOS help files are helpful!!
```
Comment (naomitress) [inline code/day2/GlatosCode.R:42]: glatos not GLATOS

File: setup.md [hunk 8]
```diff
@@ -57,8 +57,9 @@ devtools::install_github("rossdwyer/VTrack")
 
 # GLATOS - acoustic telemetry package that does filtering, vis, array simulation, etc.
```
Comment (naomitress) [inline setup.md:58]: glatos not GLATOS

**Review by naomitress — CHANGES_REQUESTED** [review]
plenty of instances where GLATOS needs to be changed to glatos, and a couple of other things to check

File: setup.md [hunk 9]
```diff
@@ -57,8 +57,9 @@ devtools::install_github("rossdwyer/VTrack")
...
 install.packages('remotes')
-library(remotes)
-install_url("https://gitlab.oceantrack.org/GreatLakes/glatos/repository/master/archive.zip",
+library(remotes) 
+install_url("https://gitlab.oceantrack.org/GreatLakes/glatos/-/raw/act-workshop-2021/glatos_0.4.2.1.tar.gz",
+
```
Comment (naomitress) [inline setup.md:62]: this line messes things up

**Review by naomitress — CHANGES_REQUESTED** [review]

File: _episodes/07-introduction-to-glatos.md (hunk 1 above)
Comment (SoftwareMonk) [reply _episodes/07-introduction-to-glatos.md:82]: Fixed

File: _episodes/08-more-features-of-glatos.md (hunk 2 above)
Comment (SoftwareMonk) [reply _episodes/08-more-features-of-glatos.md:116]: I don't think so

File: _episodes/08-more-features-of-glatos.md (hunk 3 above)
Comment (SoftwareMonk) [reply _episodes/08-more-features-of-glatos.md:56]: Fixed

File: _episodes/08-more-features-of-glatos.md (hunk 4 above)
Comment (SoftwareMonk) [reply _episodes/08-more-features-of-glatos.md:11]: Fixed

File: _episodes/09-basic-visualizations.md (hunk 5 above)
Comment (SoftwareMonk) [reply _episodes/09-basic-visualizations.md:18]: Fixed

File: _episodes/07-introduction-to-glatos.md [hunk 10]
```diff
@@ -23,41 +23,65 @@ setwd("./data")
...
 {: .language-r}
 
 Your code may not be in the 'code/glatos' folder, so use the appropriate file path for
 your data.
 
-Next, we will create paths to our detections and receiver files. glatos can
-function with both GLATOS and OTN-formatted data, but the functions are different
+
+Next, we will create paths to our detections and receiver files. GLATOS can
+function with both GLATOS and OTN Node-formatted data, but the functions are different
```
Comment (naomitress) [inline _episodes/07-introduction-to-glatos.md:35]: glatos not GLATOS

File: code/day2/GlatosCode.R (hunk 6 above)
Comment (SoftwareMonk) [reply code/day2/GlatosCode.R:7]: Gottem

File: code/day2/GlatosCode.R (hunk 7 above)
Comment (SoftwareMonk) [reply code/day2/GlatosCode.R:42]: gone-zo

File: setup.md (hunk 8 above)
Comment (SoftwareMonk) [reply setup.md:58]: confirmed

File: setup.md (hunk 9 above)
Comment (SoftwareMonk) [reply setup.md:62]: lines - 1

File: _episodes/07-introduction-to-glatos.md (hunk 10 above)
Comment (SoftwareMonk) [reply _episodes/07-introduction-to-glatos.md:35]: downsized.

**Review by naomitress — APPROVED** [review]

**Pull request merged**